### PR TITLE
Update dicom_to_nifti with a CLI

### LINKED ISSUE #86: Create CLI for dicom_to_nifti API
Suggested flags
```
-i --input
-o --output
-s --subject
-c --config
-remove_tmp
```

Motivated by #37

### LINKED ISSUE #112: Update conf/dcm2bids.json to accommodate realtime_z shimming testing data
### Description of Problem / Inefficiency:
When runnning dcm2bids with our testing data, i am running in the following issue:

<details><summary>Full details</summary>

```
dcm2bids -c ~/code/shimming-toolbox/shimming-toolbox/config/dcm2bids.json -d . -o nifti -p sub-example
INFO:dcm2bids.dcm2bids:--- dcm2bids start ---
INFO:dcm2bids.dcm2bids:OS:version: Darwin-18.7.0-x86_64-i386-64bit
INFO:dcm2bids.dcm2bids:python:version: 3.7.3 (default, Mar 27 2019, 16:54:48) [Clang 4.0.1 (tags/RELEASE_401/final)]
INFO:dcm2bids.dcm2bids:dcm2bids:version: 2.1.4
INFO:dcm2bids.dcm2bids:dcm2niix:version: v1.0.20200331
INFO:dcm2bids.dcm2bids:participant: sub-example
INFO:dcm2bids.dcm2bids:session: 
INFO:dcm2bids.dcm2bids:config: /Users/julien/code/shimming-toolbox/shimming-toolbox/config/dcm2bids.json
INFO:dcm2bids.dcm2bids:BIDS directory: /Users/julien/temp/shimming-toolbox-test/testing_data/realtime_zshimming_data/nifti
INFO:dcm2bids.utils:Running dcm2niix -b y -ba y -z y -f '%3s_%f_%p_%t' -o nifti/tmp_dcm2bids/sub-example .
INFO:dcm2bids.dcm2niix:Check log file for dcm2niix output
INFO:dcm2bids.sidecar:Sidecars pairing:
INFO:dcm2bids.sidecar:No Pairing  <-  015_._gre_field_mapping_PMUlog_2_20200817104950_e1
INFO:dcm2bids.sidecar:No Pairing  <-  015_._gre_field_mapping_PMUlog_2_20200817104950_e2
INFO:dcm2bids.sidecar:No Pairing  <-  016_._gre_field_mapping_PMUlog_2_20200817104950_e2_ph
INFO:dcm2bids.sidecar:No Pairing  <-  017_._gre_realtime_zshim_UNSHIMMED_2_20200817104950_e1
INFO:dcm2bids.sidecar:No Pairing  <-  017_._gre_realtime_zshim_UNSHIMMED_2_20200817104950_e2
INFO:dcm2bids.sidecar:No Pairing  <-  017_._gre_realtime_zshim_UNSHIMMED_2_20200817104950_e3
INFO:dcm2bids.sidecar:No Pairing  <-  034_._ep2d_bold_realtime_zshim_UNSHIMMED_2_20200817104950
INFO:dcm2bids.dcm2bids:moving acquisitions into BIDS folder

```

</details>

The dcm2bids config file should probably be updated.

Version shimming-toolbox: 589ccb8aaa2f50fafe494f970e856003acff4cfd
Version testing data: https://github.com/shimming-toolbox/data-testing/releases/tag/r20200930

### PR DESCRIPTION
**Pull request by po09i**
## Description
This PR adds a CLI for the dicom_to_nifti API. See the related issue #86 for the different options.
The PR also modifies the config file to be able to test it with realtime_zshimming_data. It adds the ability to use dicom_to_nifti with anatomical GREs and functional bold files.

TODO:
- [x] dicom_to_nifti_cli
- [x] Update config file
- [x] Create test for new config file

## Linked issues
Fixes #86 
Fixes #112

### CODE REVIEW

File: test/test_dicom_to_nifti.py [hunk 1]
```diff
@@ -6,18 +6,52 @@
 
 from shimmingtoolbox.dicom_to_nifti import dicom_to_nifti
 from shimmingtoolbox import __dir_testing__
+from shimmingtoolbox import __dir_shimmingtoolbox__
 
 
 def test_dicom_to_nifti():
     # TODO: put as decorator
```
Comment (jidicula) [inline test/test_dicom_to_nifti.py]: Could add this as a refactor issue ticket and get rid of the TODO

**Review by jidicula — CHANGES_REQUESTED** [review]
Could you run the files through a formatting tool? If you have Black installed, you can do `black -S path/to/file` for each file (-S preserves the quote style for strings).

**Comment by po09i** [comment]
> Could you run the files through a formatting tool? If you have Black installed, you can do `black -S path/to/file` for each file (-S preserves the quote style for strings).

pycharm does it automatically, I double checked and everything seems good!

**Review by jidicula — APPROVED** [review]
LGTM!

**Comment by po09i** [comment]
As discussed in internal meeting, call CLI tests cli/test_cli_XX

**Review by jidicula — APPROVED** [review]

**Pull request merged**